src/cmip6-cmor-tables/.circleci/config.yml

```diff
@@ -0,0 +1,84 @@
+version: 2
+
```

Comment:
@jkrasting the cmip6-cmor-tables directory and files should be deleted from the src directory. They were moved to the data folder in a prior commit, and it looks like this change got lost during the rebase.